dev-python/pathtools/pathtools-0.1.2-r3.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7,8} pypy)
+
+inherit distutils-r1
+
+DESCRIPTION="Pattern matching and various utilities for file systems paths"
+HOMEPAGE="https://pypi.org/project/pathtools/"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
```

Comment:
This package does have docs in the ebuild which should be built, but it is not a regression. Will merge. Please check next time to correct the previous devs lack of enabling docs.